ci-operator/step-registry/workers-rhel/pre-hook/sshtunnel/workers-rhel-pre-hook-sshtunnel-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: workers-rhel-pre-hook-sshtunnel
+  from: ansible
+  commands: workers-rhel-pre-hook-sshtunnel-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  env:
+  - name: PLATFORM_VERSION
+    documentation: |-
```

Comment:
Shall we set a default value here?

Reply:
this value is based on what rhel image used in previous provision step, and it is also defined in [workers-rhel-repos-ref](https://github.com/openshift/release/blob/master/ci-operator/step-registry/workers-rhel/repos/workers-rhel-repos-ref.yaml), which doesn't set default value, so when adding them into workflow or chain, env `PLATFORM_VERSION` need to be set anyway.